dod-advana/gamechanger-web
# Task/UOT-115295 Frontend Graph Fixes

## Description


- Icon buttons are centered around each node
- Dismiss node button now works
- Unknown document nodes no longer have a display card button

## Related Issue/Ticket




https://jira.di2e.net/browse/UOT-115289
https://jira.di2e.net/browse/UOT-115291
https://jira.di2e.net/browse/UOT-115295

## Testing instructions



Test through front end - go to any document details page and open the graph view.
1. Click on a node. Buttons should pop up in a circle centered on the node
2. Dismiss the node
3. Reset the graph. Node should come back
4. Click on an unknown document node. Display card button should not appear
